src/lib/DataTableFilterRow/DataTableFilterRow.tsx

```diff
@@ -38,6 +41,7 @@ export function DataTableFilterRow<T>({
               bsSize="sm"
               id={`filter-${column.dataField}`}
               innerRef={(ref) => ref && ref instanceof HTMLInputElement && setFilterRef(column.dataField, ref)}
+              defaultValue={predefinedFilter ? getDeepValue(predefinedFilter, column.dataField) : undefined}
```

Comment:
nitpicking: you could eliminate this duplicate line by moving it to a variable on line 32.

Reply:
32 is not possible because it depends on the column, but i was able to put it between... hope that is also fine?

Reply:
ah makes sense :-) perfect thanks!